.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: go
 go:
-- 1.4
-- 1.5
-- 1.6
-- 1.7
+- 1.9
+- 1.14
```

Comment:
Changes because vendor `golang/protobuf` imports `math/bits` (go 1.9)

5.22s$ go get github.com/golang/protobuf/proto
package github.com/golang/protobuf/proto
	imports google.golang.org/protobuf/encoding/prototext
	imports google.golang.org/protobuf/encoding/protowire
	imports math/bits: unrecognized import path "math/bits"
The command "go get github.com/golang/protobuf/proto" failed and exited with 1 during .